### PR TITLE
implicitly define properties bound to native attrs

### PR DESCRIPTION
**Pull request by garlicnation**
Fixes https://github.com/PolymerLabs/polylint/issues/90

### CODE REVIEW

File: lib/linters.js [hunk 1]
```diff
@@ -191,16 +193,28 @@ function badBindingExpressions(analyzer, is) {
...
-  var isLabel = p.hasTagName("for");
-  switch(attr.name.toLowerCase()) {
+function isNativeAttribute(name) {
+  switch (name) {
     case "class":
     case "style":
+    case "tabindex":
+    case "title":
+    case "style":
     case "href":
```
Comment (rictic) [inline lib/linters.js]: From https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes we should add:

accesskey
contenteditable
contextmenu
dir
draggable
dropzone
hidden
id
itemprop
lang
spellcheck

We should consider adding almost all of them, or keying this off of element type.

File: lib/linters.js [hunk 2]
```diff
@@ -191,16 +193,28 @@ function badBindingExpressions(analyzer, is) {
...
-function isProblematicAttribute(attr) {
-  var isLabel = p.hasTagName("for");
-  switch(attr.name.toLowerCase()) {
+function isNativeAttribute(name) {
+  switch (name) {
     case "class":
     case "style":
+    case "tabindex":
+    case "title":
+    case "style":
```
Comment (rictic) [inline lib/linters.js]: style is listed twice

File: lib/linters.js [hunk 3]
```diff
@@ -191,16 +193,28 @@ function badBindingExpressions(analyzer, is) {
...
     case "href":
       return true;
+  }
+  return false;
+}
+
+function isProblematicAttribute(attr) {
```
Comment (rictic) [inline lib/linters.js]: Recommend a comment describing what problematic means in this context

**Comment by cdata** [comment]
Does Polylint have a test suite? Asking for a friend.

**Comment by garlicnation** [comment]
@cdata https://github.com/PolymerLabs/polylint/blob/master/test/linter-test.js
I'll add a test to this PR after addressing peter's comments.

File: lib/linters.js (hunk 2 above)
Comment (justinfagnani) [inline lib/linters.js]: How about putting them in a Set?

File: lib/linters.js [hunk 4]
```diff
@@ -131,11 +132,13 @@ function attributeBindingExpressions(analyzer, is) {
   var attributes = allAttributesInDomModule(analyzer, is);
   attributes.forEach(function(attribute) {
     if (attribute.value && isBindingExpression(attribute.value)) {
+      var nativeName = attribute.name.slice(0,-1);
```
Comment (justinfagnani) [inline lib/linters.js]: why not substring? and maybe add comment that you're removing the trailing `$`

**Comment by justinfagnani** [comment]
LGTM

**Comment by sandro-k** [comment]
Thanks

**Pull request merged**